eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.11.0.0
 
+* Switch `ppCommitteeMaxTermLength` to `EpochNo`, rather than `Natural`
```

Comment:
I think this is not a usefull change.  An EpochNo is a distinct piont in time. ppCommitteeMaxTermLength is the length of an interval in this space. So if we use EpochNo we have mixed up the units.  If any thing the interval could be the underlying Num type in EpochNo which is Integer, but then we could have negative intervals, So I think Natural is the right thing.

Reply:
I agree with Tim, having it as Epoch makes it look as it could mean the Epoch representing the end of the term.

Reply:
Is there any particular reason for the change?

Reply:
I agree with the fact that this should not be an `EpochNo`, but I do not agree that it should be a `Natural`. There is a bigger reason why it has to be a narrower type, something like `Word32`, otherwise an improper PParam will lead to an integer overflow. We need a new type `EpochNoDelta` or `EpochNoInterval` or something, to indicate the intent. That being said, the reason why I switched it to `EpochNo` is for consistency with other pparams that indicate intervals in number of epochs:
  * `EMax`: https://github.com/input-output-hk/cardano-ledger/blob/cf32eab4eacebbec9817000827036822d376f35d/eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs#L183
  * `govActionLifetime`: https://github.com/input-output-hk/cardano-ledger/blob/cf32eab4eacebbec9817000827036822d376f35d/eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs#L226
  * `DRepActivity`: https://github.com/input-output-hk/cardano-ledger/blob/cf32eab4eacebbec9817000827036822d376f35d/eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs#L232
  
I've created a ticket that should take care of this properly: #3870 

I'll merge this change, just so we have at least some sort of consistency for the moment.

Reply:
makes sense